src/vendor/Github.js

```diff
@@ -0,0 +1,337 @@
+'use strict';
+
+/**
+ * The service object for handling creating contests from Github.
+ * @type {object} The Github object.
+ */
+var TC_CLIB_GITHUB = (function getGithub() {
+
+  var GITHUB_URL = 'https://api.github.com/',
+    TOKEN_KEY_GITHUB = 'glib::github_token';
+
```

Comment:
pretty much moved all the github specific code to here and made it work in it's own object.